cmd/egress/cmd.go

```diff
@@ -123,6 +126,10 @@ are set correctly before execution.
 				Proxy:        p,
 			}
 
+			if config.noTls && config.CaCert != "" {
+				fmt.Println("--cacert value will be ignored since --no-tls was passed")
+			}
```

Comment:
Can we move this check to be above line 94 (where we first check `!config.noTls`)?